docs/index.rst

```diff
@@ -24,23 +24,17 @@ Table of Contents
 =================
 
 .. toctree::
-    :maxdepth: 4
-    :numbered:
-
-    ptoled
-    keyboard
-    ptbuttons
-    pulse/eeprom
-    pulse/manual-configuration
-
-    input_components
-    output_components
-
-    component_examples
-    basic_examples
-    advanced_examples
-
-    parameters
+    :maxdepth: 2
+
+    installing
```

Comment:
Perhaps `Getting Started`? pi-topOS users won't need to do this, so we can say "use pi-topOS" before recommending manual install